rules/Php80/Rector/Class_/AttributeValueResolver.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Php80\Rector\Class_;
+
+use Nette\Utils\Strings;
+use PHPStan\PhpDocParser\Ast\PhpDoc\PhpDocTagNode;
+use Rector\Php80\ValueObject\AnnotationToAttribute;
+use Rector\Php80\ValueObject\AttributeValueAndDocComment;
+
+final class AttributeValueResolver
+{
+    /**
+     * @var string
+     * @see https://regex101.com/r/CL9ktz/1
+     */
+    private const END_SLASH_REGEX = '#\\\\\r?$#';
```

Comment:
`\r?` only without `\n` make match mac new line, if it needs not have new line after, it should restrict not followed by new line, eg: `(!?\r?\n`, I will check it.

Reply:
https://github.com/rectorphp/rector-src/pull/6579